hillstone.py

```diff
@@ -247,6 +247,13 @@ def client_info(self):
         if res[Payload.STATUS] != b'\0\0\0\0':
             raise ClientInfoError
 
+    def Int2IP(self,ipnum):
```

Comment:
You should use lowercase in its name. See Python PEP 8: Function and Variable Names.

Reply:
Also, this should be a static method. Always put whitespace after a comma.